li-hadoop-plugin/src/main/resources/dependencyPattern.json

```diff
@@ -5,7 +5,7 @@
       "message": "Open source versions(org.apache.*) of artifacts are not allowed, please use linkedin artifacts(com.linkedin.*). Open source versions will be completely blocked after March 1st 2016",
```

Comment:
I've addressed the comments and  just updated the PR. Now the complete error message looks like:

*WARN: Invalid dependency (org.apache.pig:pig:0.10.0), Open source versions of pig artifacts(org.apache.pig) are not allowed, please use linkedin artifacts(com.linkedin.pig). Open source versions will be completely blocked after March 1st 2016*